app/errors.py

```diff
@@ -0,0 +1,14 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    pass
```

Comment:
The `NotVaccinatedError` class currently uses `pass`, which means it doesn't provide any additional information or functionality. Consider adding a docstring or a custom message to improve clarity.